### ISSUE TITLE
support for other entity types

### ISSUE THREAD

**Issue by SpiGAndromeda** [karlis-vagalis/circular-timer-card#3]
Hi, this is a very nice frontend extension! I wonder if it would be possible to extend it to support numeric entities as well.

My thermostat exposes a value for the boost times that acts as a countdown. This is a just a sensor with a numeric value. It would be very nice to use that card for the visualization of that timer.

best regards,
spigandromeda

**Comment by arkixana** [comment]
Adding support for input select entities to show actual state from a repeating cycle would be nice too!!

**Comment by lcoppenr** [comment]
I agree, this extension is quite good looking.   I don't use timers much, but I have quite a few sensors that have "time remaining" values and "total time" values.  It would be great to be able to template it or add a current remaining and a total time variable.

**Comment by raphosaurus** [comment]
Oh yeah! I would love to have an input for a specifc time in hh:mm or something like that. Or for dimming your lights. And so on. More flexibility would be an absolute game changer!
